css/about.css

```diff
@@ -0,0 +1,238 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+}
+        /* End of MeyerWeb */
+        /* Begin Code */
+ p {
+     line-height: 1.4;
+}   
+
+.header nav{
+    display: flex;
+    justify-content: center;
+    align-items: baseline;
+    margin: 20px 5px 20px 5px;
+    /* border: 1px solid red; */
+}
+.header nav a{
+    margin: 30px 30px 26px 50px;
+	color: black;
+    text-decoration: none;
+}
+.nav-img{
+    margin-top: 10px;
+}
+.about-img{
+    display: flex;
+    justify-content: center;
+    width:100%;
+}
+
+.container-about-page {
+    margin:auto;
+	width: 60%;
+    text-align: left;
+}
+        /* ABOUT SECTION */
+.top-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /* font-size: 15px; */
+    margin: 40px 0 25px 0;
+    line-height: normal;
+    border-bottom: 1px dashed black;
+}
+.top-content h2{
+    font-size: 20px;
+    margin-left: 10px;
+}
+.top-content p{
+    margin-bottom: 25px;
+    margin-left: 10px;
+}
+        /* ABOUT SECTION END */
+        /* Strategy How Section Start */
+.strategy-work-container{
+    display: flex;
+    justify-content: center;
+    border-bottom: 1px dashed black;
```

Comment:
consider moving this border to the child divs instead of on the container to get that split border effect.